infra/base-images/base-builder-new/install_swift.sh

```diff
@@ -22,10 +22,10 @@ apt-get update && apt install -y $SWIFT_PACKAGES && \
   apt install -y $SWIFT_SYMBOLIZER_PACKAGES --no-install-recommends  
 
 
-wget https://swift.org/builds/swift-5.3.3-release/ubuntu1604/swift-5.3.3-RELEASE/swift-5.3.3-RELEASE-ubuntu16.04.tar.gz
-tar xzf swift-5.3.3-RELEASE-ubuntu16.04.tar.gz
-cp -r swift-5.3.3-RELEASE-ubuntu16.04/usr/* /usr/
-rm -rf swift-5.3.3-RELEASE-ubuntu16.04.tar.gz
+wget https://swift.org/builds/swift-5.3.3-release/ubuntu2004/swift-5.3.3-RELEASE/swift-5.3.3-RELEASE-ubuntu20.04.tar.gz
```

Comment:
Swift is still installed in an image that is pinned to xenial. So we don't want to upgrade this yet.

Reply:
Ok, closing